### PR TITLE
Feature/update color theme and support material3

### PR DESCRIPTION
**Pull request by alesiaradkevich**

### CODE REVIEW

File: lib/src/presentation/pages/widgets/suggestions_elevated_button.dart [hunk 1]
```diff
@@ -28,40 +28,22 @@ class SuggestionsElevatedButton extends StatefulWidget {
...
+    return SizedBox(
+      height: Dimensions.buttonHeight,
+      child: ElevatedButton(
+        style: (context.theme.elevatedButtonTheme.style ?? const ButtonStyle())
+            .copyWith(
+          backgroundColor: MaterialStatePropertyAll(widget.backgroundColor),
+          foregroundColor: MaterialStatePropertyAll(widget.textColor),
+        ),
+        onPressed: () =>
+            widget.isLoading ? () => <dynamic, dynamic>{} : widget.onClick(),
```
Comment (KirillSergeevich) [inline lib/src/presentation/pages/widgets/suggestions_elevated_button.dart:42]: maybe we can do `widget.isLoading ? () {} : widget.onClick()`

File: lib/src/presentation/pages/theme/suggestions_theme.dart [hunk 2]
```diff
@@ -1,43 +1,18 @@
...
 import 'package:suggest_a_feature/src/presentation/di/injector.dart';
-import 'package:suggest_a_feature/src/presentation/utils/font_sizes.dart';
 
 SuggestionsTheme get theme => i.theme;
 
 class SuggestionsTheme {
-  final Color primaryBackgroundColor;
-  final Color secondaryBackgroundColor;
-  final Color thirdBackgroundColor;
   final Color bottomSheetBackgroundColor;
```
Comment (KirillSergeevich) [inline lib/src/presentation/pages/theme/suggestions_theme.dart:7]: I am not sure that we need it. Let's mark it as deprecated with description to use the property from the material theme

**Review by KirillSergeevich — CHANGES_REQUESTED** [review]

File: lib/src/presentation/pages/theme/suggestions_theme.dart [hunk 3]
```diff
@@ -1,43 +1,18 @@
 import 'package:flutter/material.dart';
 import 'package:suggest_a_feature/src/presentation/di/injector.dart';
-import 'package:suggest_a_feature/src/presentation/utils/font_sizes.dart';
 
 SuggestionsTheme get theme => i.theme;
```
Comment (KirillSergeevich) [inline lib/src/presentation/pages/theme/suggestions_theme.dart:4]: we use InheritedWidget, right?

File: lib/src/presentation/pages/widgets/suggestions_elevated_button.dart (hunk 1 above)
Comment (alesiaradkevich) [reply lib/src/presentation/pages/widgets/suggestions_elevated_button.dart:42]: Yes, we can

File: lib/src/presentation/pages/theme/suggestions_theme.dart (hunk 3 above)
Comment (alesiaradkevich) [reply lib/src/presentation/pages/theme/suggestions_theme.dart:4]: For SuggestionsTheme we don't

Comment (KirillSergeevich) [reply lib/src/presentation/pages/theme/suggestions_theme.dart:4]: got you, let's use it for SuggestionsTheme too.

File: lib/src/presentation/pages/theme/suggestions_theme.dart (hunk 2 above)
Comment (KirillSergeevich) [reply lib/src/presentation/pages/theme/suggestions_theme.dart:8]: So, let's delete this property too, as we don't use it

**Pull request merged**